app/javascript/__tests__/pages/SignIn.test.tsx

```diff
@@ -10,15 +14,46 @@ jest.mock("react-helmet-async", () => {
   }
 })
 
-jest.spyOn(window, "alert").mockImplementation(() => {})
+jest.mock("@bloom-housing/ui-components", () => ({
+  ...jest.requireActual("@bloom-housing/ui-components"),
+  debounce: (fn) => fn,
+}))
+
+jest.mock("../../api/apiService", () => ({
+  post: jest.fn(),
+}))
+
+jest.mock("@bloom-housing/ui-seeds", () => {
+  const originalModule = jest.requireActual("@bloom-housing/ui-seeds")
+
+  const MockDialog = ({ children, isOpen }) =>
+    isOpen ? <div data-testid="modalMock">{children}</div> : null
```

Comment:
can you just add the props here instead of disabling the error? children are React.ReactNode and isOpen is boolean right?